alluxio-ui/common/src/components/FileView/FileView.test.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+import {configure, mount, ReactWrapper, shallow, ShallowWrapper} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import {StaticRouter} from 'react-router';
+import sinon from 'sinon';
+
+import {FileView, IFileViewProps} from './FileView';
+import {IFileBlockInfo, IFileInfo} from '../../constants';
+
+configure({adapter: new Adapter()});
+
+describe('FileView', () => {
+  let props: IFileViewProps;
+
+  beforeAll(() => {
+    props = {
+      beginInputHandler: sinon.spy(() => {
+      }),
+      endInputHandler: sinon.spy(() => {
+      }),
+      lastFetched: {},
+      offsetInputHandler: sinon.spy(() => {
+      }),
+      queryStringPrefix: '',
+      queryStringSuffix: '',
+      viewData: {
+        'currentPath': '',
+        'debug': true,
+        'fatalError': '',
+        'fileData': '',
+        'fileInfos': [],
+        'invalidPathError': '',
+        'ntotalFile': 0,
+        'viewingOffset': 0
+      }
+    };
+  });
```

Comment:
Does it make sense to test with non empty data as well?

Reply:
Yes, absolutely, this is in the to-do pile